conans/server/rest/bottle_plugins/version_checker.py

```diff
@@ -13,55 +13,34 @@ class VersionCheckerPlugin(object):
     name = 'VersionCheckerPlugin'
```

Comment:
Probably this plugin deserves a rename now, something like "ServerCapabilitiesPlugin", I guess.

Also, as the server capabilities are only used in the ``ping`` method, is there any reason why this is always applied, to all methods, instead of just returning them in the  ``ping`` endpoint? I guess there might be a reason, but I can't recall which, we'd need @lasote feedback here. Also to know if Artifactory behavior is the same.

Reply:
In Artifactory some of the methods (not all because I bet we forgot to decorate more methods) are returning the headers at any request. When we used the version information we read it from every request we made. From some versions ago it is not needed anymore so I would be ok if the server only returns the info in the `ping`.

Reply:
So for me, yes, go ahead and remove this plugin in favor of just tunning the ping endpoint.

Reply:
I've tuned the _PingController_, it is the only one returning the server capabilities.